src/utils/pdfUtils.js

```diff
@@ -0,0 +1,107 @@
+import { jsPDF } from 'jspdf';
+
+const setHeaderFont = (doc) => {
+  doc.setFontSize(14);
+  doc.setFont('Times', 'normal');
+  return doc;
+};
+
+const setTaskFont = (doc) => {
+  doc.setFontSize(14);
+  doc.setFont('Times', 'italic');
+  return doc;
+};
+
+const setSmallFont = (doc) => {
+  doc.setFontSize(12);
+  doc.setFont('Times', 'normal');
+  return doc;
+};
+
+const abbreviate = (string, length) =>
+  string?.length > length ? `${string.substring(0, length - 3)}...` : string;
+
+const generateSprintPDF = (sprint) => {
+  let depth = 0;
+  // Create PDF object and get dimensions for formatting
+  const doc = new jsPDF();
+  //const pageHeight = doc.internal.pageSize.height || doc.internal.pageSize.getHeight();
+  // Create postiion consts using doc specs
+
+  const pageWidth = doc.internal.pageSize.width || doc.internal.pageSize.getWidth();
+  const leftIndent = 20;
+  const assigneeIndent = pageWidth - 65;
+  const workedIndent = pageWidth - 40;
+  const remainingIndent = pageWidth - 20;
+
+  // Print title
+  doc.setFontSize(26);
+  doc.setFont('Times', 'bold');
+  doc.text(`Sprint ${sprint.iteration} Overview`, pageWidth / 2, (depth += 30), {
+    align: 'center',
+  });
+
+  // Print logo
+  const img = new Image();
```

Comment:
Does the image show below the title?

Reply:
Yeah it was a design choice but if you hate it I'm not married to it haha

Reply:
[K4rUd42z3YI37ggX6JBcSprint1 (25).pdf](https://github.com/hunter-bennett28/SprintCompass/files/6268025/K4rUd42z3YI37ggX6JBcSprint1.25.pdf)
ends up looking like this

Reply:
Yeah after looking at your pdf it looks beaut. Forgot how it looked from the test pdf  you sent out earlier